### DISCUSSION TITLE
Is this Line chart possible ?

### DISCUSSION THREAD

**Discussion by LucaCoduriV** [patrykandpatrick/vico#732] [Questions]
Hello, I need to do something that looks similar to this: 
![image](https://github.com/patrykandpatrick/vico/assets/43602144/9555c62c-acb9-47d5-b474-ae5138470f77)

Is this possible with your library ? If yes, how would it be done ?

Just in case in this example the background color goes from one line to an other line.

thank you

**Comment by Gowsky** [comment]
Hello! Please see #460.
